src/business-logic/facades/service-facade.ts

```diff
@@ -82,6 +83,13 @@ export class ServiceFacade {
     )
   }
 
+  public static createMongoDatabaseChangeService(): DataChangeService {
```

Comment:
We are in the business logic layer. We shouldn't have any references to Mongo here.
Also, this is a facade. Facade methods should not reference specific implementations.

Reply:
Absolutely right, this didn't get caught in my renaming after our initial talk about it! Good catch!

Reply:
Renamed to fit the service it creates!